src/secp256k1.c

```diff
@@ -64,6 +64,25 @@ static secp256k1_context secp256k1_context_no_precomp_ = {
 };
 secp256k1_context *secp256k1_context_no_precomp = &secp256k1_context_no_precomp_;
 
+size_t secp256k1_context_prealloc_size(unsigned int flags) {
```

Comment:
This needs a corresponding entry in `include/secp256k1.h`.

Reply:
Yep, "exporting functions" is on the todo list. :)